res/values-fa/strings.xml

```diff
@@ -392,7 +392,7 @@
 	<string name="no_open">هیچ فایلی باز نیست!</string>
 	<string name="copy_success">کپی شد!</string>
 	<string name="deletion">حذف %s</string>
-	<string name="extract">استخراج</string>
+	<string name="extract">استخراج apk</string>
```

Comment:
The "Extract" line is used not only for apk, but also for extracting from an archive. I recommend using some kind of generic name.